include/flash_map_pm.h

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#if USE_PARTITION_MANAGER
+#include <pm_config.h>
+#include <sys/util.h>
+
+/* Aliases for zephyr - mcuboot/ncs style naming */
+#define image_0 mcuboot_primary
+#define image_1 mcuboot_secondary
+
+#if (CONFIG_SETTINGS_FCB || CONFIG_SETTINGS_NVS)
+#define storage settings_storage
+#elif CONFIG_FILE_SYSTEM_LITTLEFS
+#define storage littlefs_storage
+#elif CONFIG_NVS
+#define storage nvs_storage
+#endif
+
+#if ((CONFIG_SETTINGS_FCB || CONFIG_SETTINGS_NVS) &&\
+     (CONFIG_FILE_SYSTEM_LITTLEFS || CONFIG_FILE_SYSTM)) ||\
```

Comment:
```suggestion
     (CONFIG_FILE_SYSTEM_LITTLEFS || CONFIG_FILE_SYSTEM)) ||\
```

Also double check the alignment.

Reply:
Alignment reflect brackets nesting.

Reply:
@hakonfam I removed constrains I had added here befor. User indedd can use these storages/FS simultaneousl if original area names are used (so not `storage` but `settings_storage`, and `nvs_stoage`).

I also that even current limit is not really required.
We might add info of zephyr sample limitation to the known-issue instead.
cc @mbolivar-nordic